File: python-sdk/src/boxsdk/models/operations/get_metadata_templates_enterprise_securityclassification_6vmvochwuwo_schema.py
import dataclasses
from typing import Optional
from boxsdk import utils
from ..shared import classificationtemplate as shared_classificationtemplate
from ..shared import clienterror as shared_clienterror


@dataclasses.dataclass
class GetMetadataTemplatesEnterpriseSecurityClassification6VMVochwUWoSchemaRequest:
    retries: Optional[utils.RetryConfig] = dataclasses.field(default=None)
    

@dataclasses.dataclass
class GetMetadataTemplatesEnterpriseSecurityClassification6VMVochwUWoSchemaResponse:
    content_type: str = dataclasses.field()
    status_code: int = dataclasses.field()
    classification_template: Optional[shared_classificationtemplate.ClassificationTemplate] = dataclasses.field(default=None)
    client_error: Optional[shared_clienterror.ClientError] = dataclasses.field(default=None)
    

File: ts-sdk/src/sdk/models/operations/getmetadatatemplatesenterprisesecurityclassification6vmvochwuwoschema.ts
import { SpeakeasyMetadata, SpeakeasyBase, RetryConfig } from "../../../internal/utils";
import * as shared from "../shared";



export class GetMetadataTemplatesEnterpriseSecurityClassification6VmVochwUWoSchemaRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  retries?: RetryConfig;
}


export class GetMetadataTemplatesEnterpriseSecurityClassification6VmVochwUWoSchemaResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  classificationTemplate?: shared.ClassificationTemplate;

  @SpeakeasyMetadata()
  clientError?: shared.ClientError;

  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;
}
